From c2afe4c901028062eebc2beb634493e69311b192 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Thu, 6 Sep 2018 18:40:34 +0000
Subject: [PATCH] run gtk-builder-convert using python2

---
 gtk/gtk-builder-convert | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtk-builder-convert b/gtk/gtk-builder-convert
index ea737de..f7c7b8f 100755
--- a/gtk/gtk-builder-convert
+++ b/gtk/gtk-builder-convert
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # Copyright (C) 2006-2008 Async Open Source
 #                         Henrique Romano <henrique@async.com.br>
-- 
2.18.0

